client-tools/Shell/TelemetricsHandler.cpp

```diff
@@ -136,6 +136,19 @@ void TelemetricsHandler::fetchTelemetricsFromServer() {
             _telemetricsFetchResponse.is(TRI_ERROR_HTTP_FORBIDDEN) ||
             _telemetricsFetchResponse.is(TRI_ERROR_HTTP_ENHANCE_YOUR_CALM)) {
           _telemetricsFetchedInfo.add(response->getBodyVelocyPack()->slice());
+          auto deploymentSlice =
+              _telemetricsFetchedInfo.slice().get("deployment");
+          if (!deploymentSlice.isNone()) {
+            auto deploymentType = deploymentSlice.get("type").stringView();
+            if (deploymentType == "active_failover") {
+              if (deploymentSlice.get("active_failover_leader").getBoolean() ==
```

Comment:
This line assumes that the attribute `active_failover_leader` is always present. This is not downwards-compatible when running the code in a mixed cluster (during an upgrade).
I think we need to check if the attribute is present, and only then call `getBoolean()` on it.